packages/teleport/src/Account/ManageDevices/AddDevice/AddDevice.tsx

```diff
@@ -38,9 +38,12 @@ import {
   requiredField,
 } from 'shared/components/Validation/rules';
 import FieldSelect from 'shared/components/FieldSelect';
+
+import createMfaOptions, { MfaOption } from 'shared/utils/createMfaOptions';
```

Comment:
I think Lisa was onto something. 🤔 Why is there a new line between line 42 and 40?

Reply:
I think this newline might be the result of using `always-and-inside-groups` instead of `always`.